### PR TITLE
Dor 73 export contact info

### PR DESCRIPTION
**Pull request by rhague-hmrc**

### CODE REVIEW

**Review by Tasmine10ten — COMMENTED** [review]
Some relatively minor comments, on the whole not too bad. 

As you're aware there was some discussion yesterday around validation on the phone number if we want to set it to accept only 11 digits or if we want to implement a way of converting it to 11 etc. This is a discussion we can have together once you're back. 

One other thing I did notice was that we allow 10 digit numbers through in **some** cases (ie. 1234567890) - appreciate you're using the recommended design option for the phone number validation/helper so I'm not sure if that's to do with that or again if it's something we'll need to address (as part of the above). Also allows some special characters so will need something to prevent that

Appreciate the validation being applied is the same for export as for import but I wonder if we'd want some tests in the **form field mappings spec** for this story as well? I do get it's essentially testing the same thing however it might be good just to have some coverage on the export side too? 


As with imports it would be great to get the email address field mandatory as part of this story however as long as it gets done as part of this sprint I won't be too fussed

File: conf/messages.en [hunk 1]
```diff
@@ -263,6 +263,8 @@ form.export-questions.vessel-details.timeOfArrival=Time of arrival
 form.export-questions.vessel-details.vesselNameOptional=Vessel name (optional)
 form.export-questions.vessel-details.dateOfArrivalOptional=Date of arrival (optional)
 form.export-questions.vessel-details.timeOfArrivalOptional=Time of arrival (optional)
+form.export-questions.contactEmail=Email address
```
[the remark this answers is not in the document]
Comment (Tasmine10ten) [reply conf/messages.en:266]: Error message for phone number -> Phone number must contain numbers only
(Will also need updated for the import error message, sorry - I missed this last time)

**Comment by arturopala** [comment]
rebased and force-pushed to resolve conflicts

**Comment by arturopala** [comment]
@Tasmine10ten this should be now ready for review again

**Review by Tasmine10ten — APPROVED** [review]
Approved on the basis that validation will be implemented as part of DOR-82 & to free up next story

**Pull request merged**